src/psyclone/transformations.py

```diff
@@ -3810,9 +3816,14 @@ def validate(self, node, options=None):
         :raises TransformationError: if the supplied node is not a CodedKern.
         :raises TransformationError: if this transformation is not applied to \
             a Gocean API Invoke.
+        :raises TransformationError: if the supplied kernel contains wildcard \
+            imports of symbols from one or more containers (e.g. a USE without\
+            an ONLY clause in Fortran).
         '''
         from psyclone.psyGen import CodedKern
         from psyclone.gocean1p0 import GOInvokeSchedule
+        from psyclone.psyir.symbols import SymbolError
```

Comment:
This is imported twice. Could you import it at the start of the file (assuming it does not cause circular dependencies).

Reply:
The earlier import had disappeared when I came to look at it but I've moved this one to the start of the file now.